src/components/network/equipment-table-editors.js

```diff
@@ -19,7 +19,8 @@ export const TapChangerSelector = ({
     tapChanger,
     setcolerror,
     resetcolerror,
-    datakey,
+    forcecolupdate,
```

Comment:
We are used to reading and writing camelCase or PascalCase. Could you emboss these identifiers ?
Alt-Shift-R could help

Reply:
I turned all these names into camelCase